### DISCUSSION TITLE
Updated Python from 3.10.7 to 3.10.8, Library not loading

### DISCUSSION THREAD

**Discussion by yianzeng** [firedrakeproject/firedrake#2629] [Firedrake support] [answered]
When running anything I receive the following error message is:

dyld[97937]: Library not loaded: /opt/homebrew/Cellar/python@3.10/3.10.7/Frameworks/Python.framework/Versions/3.10/Python
  Referenced from: <9CE5443F-0997-3128-BC1A-6DEEFF1EFF44> /Users/yianzeng/Desktop/coding/firedrake/bin/python3
  Reason: tried: '/opt/homebrew/Cellar/python@3.10/3.10.7/Frameworks/Python.framework/Versions/3.10/Python' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/python@3.10/3.10.7/Frameworks/Python.framework/Versions/3.10/Python' (no such file), '/opt/homebrew/Cellar/python@3.10/3.10.7/Frameworks/Python.framework/Versions/3.10/Python' (no such file), '/Library/Frameworks/Python.framework/Versions/3.10/Python' (no such file), '/System/Library/Frameworks/Python.framework/Versions/3.10/Python' (no such file, not in dyld cache)
zsh: abort      firedrake-update --slepc

It seems to still be looking for python version 3.10.7 which doesn't exist - is there a way to change this? I tried to change it on pyvenv.cfg but the issue persists.

**Comment by connorjward** [comment, marked as answer]
This is a known issue with homebrew. We have a suggested fix [here](https://github.com/firedrakeproject/firedrake/wiki/Fixing-venv-on-macOS-after-homebrew-patches-Python) but unfortunately if that fails we recommend a fresh installation.
Some users prevent this from recurring with [pyenv](https://github.com/pyenv/pyenv).

**Reply by yianzeng** [reply]
This worked thank you!
